src/components/Search.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Search = ({
+  placeholder,
+  withIcon,
+  withIconClickable,
+  modifier,
+  href,
+  onChange
+}) => {
+  if (withIconClickable) {
+    return (
+      <div className={`uk-${modifier === 'navbar' ? 'navbar-item' : 'margin'}`}>
```

Comment:
navbar-item should not be in this component. This component just focus on the "uk-search" component